hardware-types/pcb/rfq.json

```
@@ -0,0 +1,462 @@
+{
+    "type": "object",
+    "required": [
+        "name",
+        "manufacture"
+    ],
+    "properties": {
+        "name": {
+            "$id": "#/properties/name",
+            "type": "string",
+            "default": ""
+        },
+        "hwType": {
+            "type": "string"
```

@rcooke-warwick should this be just a type of string or an ```enum```?


```suggestion
"enum": [
                                        "PCB",
                                        "Enclosure"
                                    ],
                                    "type": "string"
```